docs/languages/polyglot.md

```diff
@@ -0,0 +1,100 @@
+# Polyglot Notebooks
+
+## What is Polyglot Programming?
+
+Polyglot programming is the practice of using multiple programming languages to leverage the strength of each language for different tasks.
+
+## What are Notebooks?
+Notebooks are interactive files that allow the mixing of executable code, visualizations, equations and narrative text. Notebooks are composed of code cells that make it easy to quickly iterate on code. Created and popularized by the open-source project [Jupyter](https://jupyter.org/), they have become the de facto tool for data science and a great resource for teaching or learning a new programming language and quick prototyping.
+
+
+
+## Polyglot Notebooks
+
+When polyglot programming meets notebooks, you get Polyglot Notebooks! As opposed to traditional notebooks that are typically used with Python, the [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode) allows you to use multiple programming languages natively all in the same notebook! No more needing
```

Comment:
It's definitely interesting, given teh comparison with Jupyter, that I do not see Python in the list of supported languages :) Do we want to perhaps say here as well that we plan to bring in more languages in the near future.

Reply:
These are the official docs so I would leave as is for now and add when it is supported. This would be more appropriate to mention in the blog post (which was added)